### PR TITLE
Adding PyInquirer

### PR DESCRIPTION
**Pull request by apeltzer**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by apeltzer** [comment]
@ewels for new nf-core tools release :)

**Comment by apeltzer** [comment]
@conda-forge-admin, please ping python

File: recipes/pyinquirer/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 24297de0dea639d8427447753d9e4c7e584e0fdb048bf61703de9ff1f28394b2
+
+
+build:
+  noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
```
Comment (synapticarbors) [inline recipes/pyinquirer/meta.yaml:16]: Remove comments

File: recipes/pyinquirer/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - python
+    - pip
+  run:
+    - python
+    - prompt_toolkit ==1.0.14
+    - Pygments >=2.2.0
+    - regex >=2016.11.21
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
```
Comment (synapticarbors) [inline recipes/pyinquirer/meta.yaml:32]: remove comments

File: recipes/pyinquirer/LICENSE.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
+Copyright 2018 Oyetoke Toby and contributors
```
Comment (synapticarbors) [inline recipes/pyinquirer/LICENSE.txt:1]: It looks like the license file is already included in the source distribution, so you should remove this manually packaged copy. Note the in the source distribution it's just called `LICENSE`, you will need to change the `license_file` line in the `meta.yaml`.

**Comment by apeltzer** [comment]
thanks @synapticarbors - I've adjusted according to your comments. Was my first recipe here 🎉

**Pull request merged**